sickbeard/name_parser/parser.py

```diff
@@ -22,37 +22,44 @@
 import time
 import re
 import os.path
+
 import sickbeard
 from sickbeard.name_parser import regexes
 
 from sickbeard import logger, helpers, scene_numbering, common, scene_exceptions, db
+from sickbeard.name_parser.guessit_parser import parser as guessit_parser
 from sickrage.helper.common import remove_extension
 from sickrage.helper.encoding import ek
 from sickrage.helper.exceptions import ex
 from sickbeard.helpers import remove_non_release_groups
 import dateutil
 
+pre_cleanup_re = re.compile(r'(\.vol\d+\+\d+)?((\.par2\b)|(\.nzb\b)|(\.mkv\b)).*((\d+\.of\.\d+)|(\(\d+/\d+\))).*$',
+                            flags=re.IGNORECASE)
 
```

Comment:
Sure. I'll still revise de code style for the new files and the modifications I made on this file